src/data/metadata.ts

```
@@ -0,0 +1,131 @@
+  { id: 44, sura_name: "الدخان", sura_name_en: "The Smoke", sura_name_romanization: "Ad-Dukhan'", total_verses: 59, juz_id: [25], page_id: 496 },
+  { id: 45, sura_name: "الجاثية", sura_name_en: "The Kneeling Down", sura_name_romanization: "Al-Jathiya", total_verses: 37, juz_id: [25], page_id: 499 },
+  { id: 46, sura_name: "الأحقاف", sura_name_en: "The wind-curved sandhills", sura_name_romanization: "Al-Ahqaf", total_verses: 35, juz_id: [26], page_id: 502 },
+  { id: 47, sura_name: "محمد", sura_name_en: "Muhammad", sura_name_romanization: "Muhammad", total_verses: 38, juz_id: [26], page_id: 507 },
+  { id: 48, sura_name: "الفتح", sura_name_en: "The Conquest", sura_name_romanization: "Al-FatH", total_verses: 29, juz_id: [26], page_id: 511 },
+  { id: 49, sura_name: "الحجرات", sura_name_en: "The private apartments", sura_name_romanization: "Al-Hujurat", total_verses: 18, juz_id: [26], page_id: 515 },
+  { id: 50, sura_name: "ق", sura_name_en: "The Letter Qaf", sura_name_romanization: "Qaf", total_verses: 45, juz_id: [26], page_id: 518 },
+  { id: 51, sura_name: "الذاريات", sura_name_en: "The winnowing winds", sura_name_romanization: "Adh-Dhariyat", total_verses: 60, juz_id: [26, 27], page_id: 520 },
+  { id: 52, sura_name: "الطور", sura_name_en: "Mount Sinai", sura_name_romanization: "At-Tur", total_verses: 49, juz_id: [27], page_id: 523 },
+  { id: 53, sura_name: "النجم", sura_name_en: "The Star", sura_name_romanization: "An-Najm", total_verses: 62, juz_id: [27], page_id: 526 },
+  { id: 54, sura_name: "القمر", sura_name_en: "The moon", sura_name_romanization: "Al-Qamar", total_verses: 55, juz_id: [27], page_id: 528 },
+  { id: 55, sura_name: "الرحمن", sura_name_en: "The Beneficient", sura_name_romanization: "Ar-Rahman", total_verses: 78, juz_id: [27], page_id: 531 },
```

ESLint

[error] Replace `·id:·55,·sura_name:·"الرحمن",·sura_name_en:·"The·Beneficient",·sura_name_romanization:·"Ar-Rahman",·total_verses:·78,·juz_id:·[27],·page_id:·531` with `⏎····id:·55,⏎····sura_name:·'الرحمن',⏎····sura_name_en:·'The·Beneficient',⏎····sura_name_romanization:·'Ar-Rahman',⏎····total_verses:·78,⏎····juz_id:·[27],⏎····page_id:·531,⏎·`

(prettier/prettier)